src/styles/layouts/_login.scss

```diff
@@ -3,20 +3,74 @@
   .container {
     margin: 0 auto;
     width: 600px;
+    box-shadow: 0 5px 10px 0px rgba(0, 0, 0, 0.1);
```

Comment:
Usunąłem to obramowanie, które wcześniej także dodałeś. Zauważ przy mobilkach (główny priorytet), powiedzmy szerokość okna 425px, nie wygląda to dobrze. Jeżeli koniecznie je chcesz, można pokombinować z `media query` ;)